extensions/blocks/likes/likes-checkbox.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { CheckboxControl } from '@wordpress/components';
+import { Component } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import { withDispatch, withSelect } from '@wordpress/data';
+
+class LikesCheckbox extends Component {
+	render() {
+		const { areLikesEnabled, toggleLikes } = this.props;
+		return (
+			<CheckboxControl
+				label={ __( 'Show likes.', 'jetpack' ) }
+				checked={ areLikesEnabled }
+				onChange={ value => {
+					toggleLikes( value );
+				} }
```

Comment:
I think `onChange={ toggleLikes }` should be sufficient here :slightly_smiling_face: